src/main/java/Duke.java

```diff
@@ -1,10 +1,108 @@
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        //init
+        Scanner sc = new Scanner(System.in);
+        ToDoList ls;
+
+        ls = Duke.startUp();
+        Duke.input(sc, ls);
+        Duke.shutDown(ls);
     }
+
+    private static ToDoList startUp() {
+        try {
+            String logo = " ____        _        \n"
+                    + "|  _ \\ _   _| | _____ \n"
+                    + "| | | | | | | |/ / _ \\\n"
+                    + "| |_| | |_| |   <  __/\n"
+                    + "|____/ \\__,_|_|\\_\\___|\n";
+            System.out.println("Hello from\n" + logo);
+            String divider = "____________________________________________________________\n";
+            System.out.println(divider + "What can the Duke help you with today?\n" + divider);
+            return ToDoList.load();
+        } catch(Exception e) {
+            return new ToDoList();
+        }
+    }
+
+    private static void shutDown(ToDoList ls) {
+        try {
+            ls.save();
+            String divider = "____________________________________________________________\n";
+            System.out.println(divider + "Goodbye, feel free to call the Duke again whenever you need.\n" + divider);
+        } catch(Exception e) {
+
+        }
+    }
+
+    private static void input(Scanner sc, ToDoList ls) {
+        while (true) {
+            try {
+                String[] input = Duke.commandHandler(sc.nextLine(), " ", 2, 1);
+                String command = input[0];
+                String sub;
+
+                switch (command) {
+                    case "bye":
+                        return;
+                    case "list":
+                        System.out.println(ls);
+                        break;
+                    case "mark":
+                        sub = input[1];
+                        ls.markTask(Integer.parseInt(sub)); //numbersformatexception to be handled
+                        break;
+                    case "unmark":
+                        sub = input[1];
+                        ls.unmarkTask(Integer.parseInt(sub)); //numbersformatexception to be handled
+                        break;
+                    case "delete":
+                        sub = input[1];
+                        ls.delete(Integer.parseInt(sub)); //numbersformatexception to be handled
+                        break;
+                    case "todo":
+                    case "event":
+                    case "deadline":
+                        Duke.taskCommandHandler(input, ls);
+                        break;
+                    default:
+                        throw new DukeException("The Duke does not understand your words!");
```

Comment:
As per Java coding standards, the cases should be on the same indentation level as switch.